components/AdminTable/AdminDiscussionsTable/AdminDiscussionsTable.vue

```diff
@@ -83,47 +106,58 @@
             </p>
           </div>
         </td>
+        <td>
+          {{ formatDate(discussion.created) }}
+        </td>
         <td>
           <template v-if="discussion.closed">
             {{ formatDate(discussion.closed) }}
           </template>
         </td>
         <td>
-          <a
-            v-if="subjects[discussion.subject.id]"
-            class="fr-link inline-flex"
-            :href="getDiscussionUrl(discussion.id, subjects[discussion.subject.id])"
-            target="_blank"
+          <BrandedButton
```

Comment:
Titles are necessary to make the button accessibles

Reply:
The button already has a content? `icon-only` props add a `sr-only` on the text to not show it but screen reader should be able to read it?